ui/app/scripts/container/views/containerlist.html

```diff
@@ -18,8 +18,7 @@
     </tr>
     </thead>
     <tbody>
-    <tr ng-repeat="item in containers | filter:filterQuery | orderBy: item.groups"
-        ng-show="!item.inactive">
+    <tr dir-paginate="item in pageable.items | itemsPerPage: pageable.pageSize | filter:filterQuery" total-items="pageable.total" ng-show="!item.inactive">
```

Comment:
This will make the filterQuery applicable only to the current page. I guess that is fine.